src/include/daos_types.h

```diff
@@ -273,12 +273,28 @@ typedef struct {
 	uint32_t			pi_leader;
 	/** pool info bits, see daos_pool_info_bit */
 	uint64_t			pi_bits;
+	/** Possible TODO - add number of pool service replicas?
+	 *  uint32_t			pi_nvc;
+	 */
 	/** Space usage */
 	struct daos_pool_space		pi_space;
 	/** rebuild status */
 	struct daos_rebuild_status	pi_rebuild_st;
 } daos_pool_info_t;
 
+/*
+ * DAOS management pool information
+ */
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
I would like to disregard this style warning in this patch because the newly introduced typedef / struct is in the same style as the other definitions in the file.